vcxagency-node/package.json

```diff
@@ -24,9 +24,12 @@
     "test:unit:configuration": "SILENT_WINSTON=true jest --runInBand ./test/unit/configuration",
     "test:unit:entities": "SILENT_WINSTON=true jest --runInBand ./test/unit/entities",
     "test:unit:messaging": "SILENT_WINSTON=true jest --runInBand ./test/unit/messaging",
+    "test:unit:messaging:aries": "SILENT_WINSTON=true jest --runInBand ./test/unit/messaging/aries-msgs.spec.js",
     "test:unit:messaging:agent": "SILENT_WINSTON=true jest --runInBand ./test/unit/messaging/agent-msgs.spec.js",
     "test:unit:messaging:aconn": "SILENT_WINSTON=true jest --runInBand ./test/unit/messaging/aconn-msgs.spec.js",
     "test:unit:notifications": "SILENT_WINSTON=true jest --runInBand ./test/unit/notifications",
+    "test:unit:notifications:longpoll": "SILENT_WINSTON=true jest --runInBand ./test/unit/notifications/longpoll.spec.js",
+    "test:unit:notifications:adapter": "SILENT_WINSTON=true jest --runInBand ./test/unit/notifications/event-adapter-redis.spec.js",
```

Comment:
Perhaps if the goal is to silence winston in tests, it would be better pass the parameter from the tests directly instead of "polluting" environment variable space and relying on a script clause to set it. I understand the argument of convenience of this approach, though.

Reply:
Sounds like a thing for different PR, but I get it that it just dropped in your eye here.

Reply:
Yeah, I am not insisting on doing it in this PR